spring-kafka/src/main/java/org/springframework/kafka/support/KafkaUtils.java

```diff
@@ -38,12 +38,32 @@
  *
  * @author Gary Russell
  * @author Wang ZhiYang
+ * @author Soby Chacko
  *
  * @since 2.2
  *
  */
 public final class KafkaUtils {
 
+	/**
+	 * Header name for deserialization exceptions.
+	 * @since 3.2
```

Comment:
According to the GH issue mandate this fix is supposed to be back-ported down to `3.0.x`